src/pages/patientView/PatientViewGenePanelModal/styles.scss

```diff
@@ -0,0 +1,31 @@
+#patient-view-gene-panel {
```

Comment:
lets convert this to a css module.    you can look at other styles.module.scss files for examples.  happy to help.